### DISCUSSION TITLE
QField for QGIS no longer on Play Store?

### DISCUSSION THREAD

**Discussion by OBVYamaska** [opengisch/QField#2847] [Q&A]
I hope I simply fell in a moment between updates, but the app is no longer available on Play store. I can only find the beta version, which clearly states is not intended for production use.

Most of our project using the app start this week... bad timing!

**Comment by mbernasocchi** [comment]
Hey @OBVYamaska indeed just bad timing, the app will reappear on the playstore as soon as the review process is trough. For some reasons google notified today (after 4-5 releases that included the cloud component) that we had forgotten to give them cloud test credentials.
all is done on our side and usually the review cycle is pretty fast (max half a day so far).
Meanwhile you can get qfield on https://qfield.org/releases 
cheers Marco
